GooglePlayPlugins/com.google.play.billing/Samples/.gitignore

```diff
@@ -0,0 +1,67 @@
+/[Ll]ibrary/
```

Comment:
Is this based off the standard Unity .gitignore template? https://github.com/github/gitignore/blob/master/Unity.gitignore

Reply:
yes